### PR TITLE
chore: Unify template spacing

### PR DESCRIPTION
**Pull request by ihmpavel**
### What does it do?
Unify spacing with literals in Strapi repo

### Describe the technical changes you did.
Changed every `{{variable}}` to `{{ variable }}`

### Why is it needed?
Unification of code. Some parts has variables without spaces, some has with spaces

### CODE REVIEW

**Review by ihmpavel — COMMENTED** [review]
Add forgotten spaces

File: packages/generators/generators/lib/templates/js/policy.js.hbs [hunk 1]
```diff
@@ -1,12 +1,12 @@
 'use strict';
 
 /**
- * `{{id}}` policy.
+ * `{{ id }}` policy.
```
Comment (markkaylor) [inline packages/generators/generators/lib/templates/js/policy.js.hbs:4]: could you remove the `.` here as well to be consistent with the other files?

**Review by markkaylor — CHANGES_REQUESTED** [review]
Looks good to me, I think just one small thing to fix if you don't mind

File: packages/generators/generators/lib/templates/js/policy.js.hbs (hunk 1 above)
Comment (ihmpavel) [reply packages/generators/generators/lib/templates/js/policy.js.hbs]: Removed

**Review by markkaylor — APPROVED** [review]
LGTM

**Comment by ihmpavel** [comment]
Hi, does PR wait on something I should do?

**Pull request merged**